home/tests.py

```diff
@@ -1,3 +1,51 @@
-# from django.test import TestCase
+from django.test import TestCase
+from .models import Question, Answer
+from django.test import Client
+from django.urls import reverse
 
-# Create your tests here.
+
+class QuestionModelTests(TestCase):
```

Comment:
You don't need to define a test class and certainly nit inherit from TestCase. This is how you use [unittest][1] while we instructed you to use [PyTest][2]

[1]: https://docs.python.org/3/library/unittest.html
[2]: https://docs.pytest.org/en/6.2.x/

Reply:
fixed